bigquery/tests/unit/test__pandas_helpers.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+
+try:
+    import pandas
+except ImportError:  # pragma: NO COVER
+    pandas = None
```

Comment:
If you ever get tired of the try/except pattern, you can write a maybe_import(*args) function that returns a tuple of modules:

```
def maybe_import(*args):
  modules = []
  for arg in args:
    try:
      modules.append(__import__(arg))
    except ImportError:
      return tupe([None] * len(args))
  return tuple(modules)
```